fem/hybridization.hpp

```diff
@@ -106,6 +111,32 @@ class Hybridization
    void SetConstraintIntegrator(BilinearFormIntegrator *c_integ)
    { delete c_bfi; c_bfi = c_integ; }
 
+   /** Add the boundary face integrator that will be used to construct the
+       constraint matrix C. The Hybridization object assumes ownership of the
+       integrator, i.e. it will delete the integrator when destroyed. */
+   void AddBdrConstraintIntegrator(BilinearFormIntegrator *c_integ)
+   {
+      c_bfbfi.Append(c_integ);
+      c_bfbfi_marker.Append(NULL); // NULL marker means apply everywhere
+   }
+   void AddBdrConstraintIntegrator(BilinearFormIntegrator *c_integ,
+                                   Array<int> &bdr_marker)
+   {
+      c_bfbfi.Append(c_integ);
+      c_bfbfi_marker.Append(&bdr_marker);
+   }
+
+   /// Access all integrators added with AddBdrConstraintIntegrator().
+   Array<BilinearFormIntegrator*> *GetBCBFI() { return &c_bfbfi; }
+
+   /// Access all boundary markers added with AddBdrConstraintIntegrator().
+   /** If no marker was specified when the integrator was added, the
+       corresponding pointer (to Array<int>) will be NULL. */
+   Array<Array<int>*> *GetBCBFI_Marker() { return &c_bfbfi_marker; }
```

Comment:
I realize there is a precedent in MFEM for using pointers, but you could start a trend here of returning a reference.